Makefile

```diff
@@ -27,7 +27,7 @@ fmt: install
 	@$(ESLINT) --fix .
 .PHONY: fmt
 
-test: install
+test: install lint
```

Comment:
Not sure what the best practice here is. We have lint as part of the test for client side integrations. @Peripheral1994 do we intentionally not lint as part of `make test` for server side integrations?